frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css

```diff
@@ -0,0 +1,60 @@
+.ArgumentsGrid {
```

Comment:
:x: "< Custom Expression" should be gray

![image](https://github.com/user-attachments/assets/d411c31b-fbf8-48b2-af80-bf832d95755b)

Reply:
fixed  6eb9d18

<img width="561" alt="image" src="https://github.com/user-attachments/assets/319e871b-0f8b-41e5-8923-97736dfa0642">